lib/book.rb

```diff
@@ -0,0 +1,17 @@
+require './item'
```

Comment:
[OPTIONAL] I think it would be better if you use `require_relative` to import `item`.` Require` should be used for external files, like gems, while `require_relative` should be used for referring to files within your directory.

Reply:
Thanks for the recommendation to fix it.  I have done it Thank you.